### PR TITLE
Loot updates

### PR DESCRIPTION
**Pull request by BenDriller**
June 15 2020

### CODE REVIEW

File: KTLOSBWLLoot.json [hunk 1]
```diff
@@ -185,14 +185,14 @@
...
-			"prio":["Mainspec Dagger Rogues prio","Dagger tank w/ ACLG","Offspec Rogues","EP/GP"],
+			"prio":["Ucuucu","Mainspec Dagger Rogues prio","Dagger tank w/ ACLG","Offspec Rogues","EP/GP"],
 			"gp":"",
 			"wowID":""
 		}
 	},
 	"Broodlord Lashlayer":{
 		"Boots of Transcendence":{
-			"prio":["Jowsus (1)","Claris (1)","WTBglad (3 piece)(2)","EP/GP"],
+			"prio":["Jowsus(1)","Claris(1)","WTBglad(2)","EP/GP"],
```
Comment (zhang165) [inline KTLOSBWLLoot.json:195]: Probably remove WTBGlad since he's bann and gone :P

File: KTLOSBWLLoot.json [hunk 2]
```diff
@@ -603,7 +603,7 @@
 			"wowID":""
 		},
 		"Ashkandi Greatsword of the Brotherhood":{
-			"prio":["Chrixus(1)","Oph(1)","PVP prio","Warriors, Hunter, Pally","EP/GP"],
+			"prio":["Chrixus(2)","Oph(1)","PVP prio","LC","Warriors, Hunter, Pally","EP/GP"],
```
Comment (zhang165) [inline KTLOSBWLLoot.json:606]: can we add Wizzo here? :P

File: KTLOSMCLoot.json [hunk 3]
```diff
@@ -423,6 +428,11 @@
 			"prio":["EP/GP"],
 			"gp:":"",
 			"wowID":""
+		},
+		"Bindings of the Windseeker":{
```
Comment (zhang165) [inline KTLOSMCLoot.json:432]: we should call this left and right so it's less confusing :P

File: KTLOSBWLLoot.json (hunk 2 above)
Comment (zhang165) [inline KTLOSBWLLoot.json:606]: This was also a last minute change by one James, was this discussed?  If not we should roll this back

Comment (BenDriller) [reply KTLOSBWLLoot.json]: we discussed it, we're going to keep Oph in

**Pull request merged**